mget.go

```diff
@@ -97,12 +97,22 @@ func (m MGetResult) GetSource() []byte {
 	return []byte(m.Source)
 }
 
-// ValidateForVersion checks for validation in the [MGetRequest] instance relating to the version of OpenSearch given by v.
-func (m MGetRequest) ValidateForVersion(v Version) (ValidationResults, error) {
-	switch v {
-	case V2:
-		return validateMGetRequestForV2(m), nil
-	default:
-		return nil, fmt.Errorf("Invalid version: %s", v)
+// Validate validates the given MGetRequest
+func (m *MGetRequest) Validate() ValidationResults {
+	validationResults := make(ValidationResults, 0)
```

Comment:
Nit: no need to explicitly make a 0-length slice here.  `append()` works on `nil` and returning `nil` for a slice with no values is idiomatic.

Reply:
Do we also want to return a `ValidationResults`? I would think we want `nil` if no validation issues.

Reply:
Ok, thanks!